packages/node_modules/@webex/plugin-meetings/src/meeting/request.js

```diff
@@ -61,12 +63,15 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       respOnlySdp: true,
       allowMultiDevice: true,
       ensureConversation: ensureConversation || false,
-      supportsNativeLobby: 1,
-      clientMediaPreferences: {
-        preferTranscoding: preferTranscoding ?? true
-      }
+      supportsNativeLobby: 1
     };
 
+    if (isBoolean(preferTranscoding)) {
+      body.clientMediaPreferences = {
+        preferTranscoding
```

Comment:
What does the server prefer? Null or false? We could easily default the value to `false` in the destructuring on line 45.

```js
const {preferTranscoding = false} = options;
```

Reply:
They were ok with both